lib/rules/lines-between-class-members.js

```diff
@@ -55,7 +55,30 @@ module.exports = {
          * @returns {boolean} True if there is at least a line between the tokens
          */
         function isPaddingBetweenTokens(first, second) {
```

Comment:
the function seems a little complicated for me, could you add a comment to say what was happening, thanks!

Reply:
@Aladdin-ADD Sure, I'll add comments.